src/main/java/emissary/kff/KffFile.java

```diff
@@ -164,7 +164,7 @@ private boolean binaryFileSearch(@Nullable byte[] hash, long crc) {
             // mappedBuf.position(record.length * mid);
             // mappedBuf.get(record);
             try {
-                knownFile.seek(record.length * mid);
```

Comment:
I opted to make this change instead of converting all the variables involved to long.